docker/pravega/Dockerfile

```diff
@@ -9,6 +9,9 @@
 #
 FROM java:8
 
+RUN apt-get update && apt-get install -y -q rpcbind nfs-common \
```

Comment:
I think I share the same concern as @tkaitchuck. Why should we do `apt-get` and `apt-install` here? If we do this, are we forcing everyone using Pravega with docker to install these packages? Is there any potential conflict? Is there a way of doing it via the docker compose files so that we do it only for the simple deployments we intend it for? What if it is not ubuntu?